dev-docs/modules/iabCatalogTranslation.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page_v2
+page_type: module
+title: Module - IAB Category Translation
+description: Converts between ad agency brand categories and IAB brand categories.
+module_code : CategoryTranslation
+display_name : CategoryTranslation
+enable_download : true
+sidebarType : 1
+---
+
+# IAB Category Translation
```

Comment:
Why is filename iabCatalogTranslation when it's Category Translation? (If you change the filename, but sure to search/replace all to correct all links.)

Reply:
Changed file name.